### ISSUE TITLE
A problem about TableView Cell.

### ISSUE THREAD

**Issue by haozeWang** [uchicago-mobi/MPCS51030-2017-Winter-Forum#181]
Hi, professor. I write a tableview to show all task I designed. But the content of my cell will change when scrolling, Just like when I scroll down and go back, the content in the first cell will become different.  I do not know how to solve this problem. Maybe this is a problem caused by cell reuse? 
This is my code. 

<img width="720" alt="2017-03-12 4 31 14" src="https://cloud.githubusercontent.com/assets/24993936/23836134/805f06a2-0741-11e7-96ae-1d91eeed25f3.png">

**Comment by alexliu0809** [comment]
I agree that it might have something to do with table reuse. Every time the cell will appear on the screen, it is regenerated in function "cellForRowAt". When it leaves the screen, it can no longer be referenced.

**Comment by hbennett766** [comment]
A cell reuse issue is a good guess. Can you commit your code to your repository? I'll take a look at it.
